packages/paste-core/components/disclosure/__tests__/disclosure.test.tsx

```diff
@@ -50,6 +81,17 @@ describe('Disclosure', () => {
     expect(renderedDisclosureButton.getAttribute('aria-disabled')).toEqual('true');
     expect(renderedDisclosureButton.getAttribute('tabindex')).toEqual('0');
   });
+  it('should render a disclosure open using a state hook', () => {
+    render(<StateHookMock />);
+    const renderedDisclosureButton = screen.getByRole('button');
+    expect(renderedDisclosureButton.getAttribute('aria-expanded')).toEqual('true');
+  });
+  it('should update attributes when clicked using a state hook', () => {
+    render(<StateHookMock />);
+    const renderedDisclosureButton = screen.getByRole('button');
+    fireEvent.click(renderedDisclosureButton);
```

Comment:
Prefer merging this with the previous `it()` call, to avoid having to render the same mock twice.